### PR TITLE
Added pop-up for comment 

### PR DESCRIPTION
**Pull request by tanujvyas10**
Fixes Issue #399 
ezgif com-video-to-gif (5)

### CODE REVIEW

**Comment by tanujvyas10** [comment]
@vaibhavdaren please review

File: src/user/dashboard/news-feed/popups/comment.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+        super(props);
+    }
+    render(){
+    return (
+        <Modal
+      {...this.props}
+      size="md"
+      aria-labelledby="contained-modal-title-vcenter"
+      centered>
+      <div className="container">
```
Comment (vaibhavdaren) [inline src/user/dashboard/news-feed/popups/comment.js:18]: load data from json.  in exports. and render the data

Comment (tanujvyas10) [reply src/user/dashboard/news-feed/popups/comment.js:18]: very well sir

Comment (tanujvyas10) [reply src/user/dashboard/news-feed/popups/comment.js:18]: @vaibhavdaren sorry so say, I did not understand your concern. Can you please be more specific regarding the area that needs changes

Comment (vaibhavdaren) [reply src/user/dashboard/news-feed/popups/comment.js:18]: Yes have a file named data.json and render the array as if you are getting a response from API.
This is called data mocking. This practice is used when the frontend development is ahead of backend development in agile software development life cycle.

Comment (tanujvyas10) [reply src/user/dashboard/news-feed/popups/comment.js:18]: Sure, Thank you , i will update it soon

**Comment by tanujvyas10** [comment]
@vaibhavdaren Now the comments are rendering from JSON-data/comments
Please review the work

**Pull request merged**